runtime/j9vm/jvm.c

```diff
@@ -4425,25 +4433,32 @@ isSignalSpecial(jint sigNum)
 #if defined(SIGINT)
 		(SIGINT == sigNum) ||
 #endif /* defined(SIGINT) */
-#if defined(SIGQUIT)
-		(SIGQUIT == sigNum) ||
-#endif /* defined(SIGQUIT) */
 #if defined(SIGTERM)
 		(SIGTERM == sigNum) ||
 #endif /* defined(SIGTERM) */
 		FALSE;
 }
 
 /**
- * Check if a signal is used by the VM.
+ * Check if a signal is reserved by the VM.
+ *
+ * Synchronous signals are handled by the VM via omrsig_protect: SIGFPE,
+ * SIGILL, SIGSEGV, SIGBUS and SIGTRAP. On Windows, SIGBREAK is reserved
+ * by the VM to trigger dumps. On Unix/Linux platforms, SIGQUIT is
+ * reserved by the VM to trigger dumps. SIGABRT is reserved by the JVM
+ * for abnormal termination. SIGCHLD is reserved for internal control.
+ * On zOS, SIGUSR1 is reserved by the JVM. SIGRECONFIG is reserved to
+ * detect any change in the number of CPUs, processing capacity, or
+ * physical memory. This function only needs to ignore signals included
+ * in jvm.c::signalMap.
```

Comment:
> This function only needs to ignore signals included in jvm.c::signalMap.

@babsingh I'm not sure what this comment means.  Does ignore mean return false?

Reply:
i will update the comment. i meant to say the following:

*if the `signal` is listed in `jvm.c::signalMap`, then only `isSignalReservedByJVM` needs to determine if the `signal` is used/reserved by the JVM. if the signal is not listed in `jvm.c::signalMap`, then a `sun.misc.Signal` instance can't be created for a `signal` value. The native `JVM_*Signal` functions can't be invoked in the absence of a `sun.misc.Signal` instance.*

Reply:
comment has been updated.

Reply:
Thanks.  That's much clearer